datalad/core/local/tests/test_save.py

```diff
@@ -643,6 +643,9 @@ def test_partial_unlocked(path):
     ds.save()
     assert_repo_status(ds.path)
     # but now a change in the attributes
+    if ds.repo.git_annex_version >= '10.20220127':
+        # see gh-6570
+        raise SkipTest("annex bug https://git-annex.branchable.com/bugs/Change_to_annex.largefiles_leaves_repo_modified/")
```

Comment:
claims to be fixed as of 3 hours ago https://git-annex.branchable.com/bugs/Change_to_annex.largefiles_leaves_repo_modified/

Reply:
Thx. Updated #6570 and will push a change to narrow the version range for this skip.